lib/utils.ts

```diff
@@ -16,3 +15,17 @@ export function autoGrow(ref: React.RefObject<HTMLTextAreaElement>) {
     console.warn("autoGrow: ref is not set or current is null");
   }
 }
+
+//  Automatically adjusts the height of a textarea based on its content.
+
+export function autoGrow(ref: React.RefObject<HTMLTextAreaElement>) {
+  if (ref.current) {
+    const { current } = ref;
+    current.style.height = "auto";
+    current.style.height = `${current.scrollHeight}px`;
+  } else {
+    console.warn("autoGrow: ref is not set or current is null");
+  }
+}
+
+export const baseURL = process.env.NEXT_PUBLIC_URL || "http://localhost:3000"
```

Comment:
and explain me the need of this

Reply:
This is because when we fetch data from routes other then `/` like In this case fetching from `/profile`.
So in fetch we need to specify full URL `http://localhost:3000/api/getUserProfile` at the place of only `/api/getUserProfile`